packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -272,7 +272,8 @@ export default function VisualEditor( { styles } ) {
 		{
 			'is-layout-flow': ! themeSupportsLayout,
 		},
-		themeSupportsLayout && postContentLayoutClasses
+		themeSupportsLayout && postContentLayoutClasses,
+		align && `align${ align }`
```

Comment:
I know this is likely pretty edge-casey, but what about the case where the wide post content block has the constrained layout type (or `inherit: true`)? In that case, in the site editor, the preview of the post content block will visually look like the content is constrained to the contentSize, but when opening up the page editor, the `alignwide` rules take precedence.

To avoid that, I was wondering if we could add a variable to use before outputting the classname and styles. E.g. something along the lines of:

```
const { layout = {}, align = '' } = newestPostContentAttributes || {};
const outputAlignmentStyles = !! ( align && ( ! layout.type || layout.type === 'default' ) );
```

And then use that in this line and before outputting `<LayoutStyle...` below?

Reply:
Well spotted! I think we should still output the alignment styles though, because if the Post Content is alignwide none of its children will ever be wider than that, so it helps to have the content max width set. 

Instead, I'm going to try scoping the rules affecting Post Content children to when Post Content itself is `is-layout-flow`.

Reply:
> Instead, I'm going to try scoping the rules affecting Post Content children to when Post Content itself is is-layout-flow.

Nice, that helps keep the code simpler, too 👍